sonic-net/sonic-buildimage
# [BFN] Stop PMON before swss during warm reboot

Signed-off-by: Andriy Kokhan <andriyx.kokhan@intel.com>

#### Why I did it
Stopping of pmon after swss and syncd causes some ERROR logs in syslog. Also, this affects teamd downtime.

#### How I did it
Adjust warmboot shutdown order in make file

#### How to verify it
Build SONiC image, deploy to the target device and check `/etc/sonic/warm-reboot_order` content.
```
lldp mux nat radv sflow bgp pmon swss teamd syncd
```

#### Which release branch to backport (provide reason below if selected)
